src/components/Tabs/Tabs.tsx

```diff
@@ -65,66 +61,37 @@ export const Tabs = ({
   onChange,
   ...other
 }: Props) => {
-  const ref = useRef<number | undefined>();
+  const getUID = useUIDSeed();
+  const activeTabPanelId = useUID();
```

Comment:
Only maintaining a single id for the active tab panel, instead of ids for every tab panel.

Since we only actually render one tab panel at a time (the active one), I think this makes sense. Better off not referencing ids that aren't actually in the DOM.